webui.py

```diff
@@ -1042,8 +1107,9 @@ def img2img(prompt: str, image_editor_mode: str, init_info, mask_mode: str, mask
     sort_samples = 6 in toggles
     write_info_files = 7 in toggles
     jpg_sample = 8 in toggles
-    use_GFPGAN = 9 in toggles
-    use_RealESRGAN = 10 in toggles
+    range_based_gen = 9 in toggles
+    use_GFPGAN = 10 in toggles
+    use_RealESRGAN = 10 in toggles if GFPGAN is None else 11 in toggles # possible index shift

```

Comment:
same as above
remember: if in doubt, don't mess, you could have just added your option to the end